src/components/navbar/SideBar.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import chessIcon from "../../assets/navbar/Chess.svg";
+import gameControllerIcon from "../../assets/navbar/GameController.svg";
+import houseIcon from "../../assets/navbar/House.svg";
+import infoIcon from "../../assets/navbar/Info.svg";
+import lockKeyIcon from "../../assets/navbar/LockKey.svg";
+import moonStarIcon from "../../assets/navbar/MoonStars.svg";
+import questionIcon from "../../assets/navbar/Question.svg";
+import blankProfile from "../../assets/navbar/blankProfile.svg";
+import Page from "./Page";
+
+interface props {
+  page: Page;
+}
+
+const SideBar: React.FC<props> = ({ page }) => {
+  let color;
+  switch (page) {
+    case Page.MAIN:
+      color = "bg-primary";
+      break;
+    case Page.ADMIN:
+      color = "bg-secondary";
+      break;
+    case Page.FORTUNE:
+      color = "bg-johnjud-purple";
+      break;
+  }
+
+  return (
+    <div className={`h-full w-44 p-4 pt-8 text-white ${color}`}>
```

Comment:
ขอ padding left กับ right กว้างกว่านี้หน่อย จริงๆลอง p-8 น่าจะโอเค

Reply:
เอา width ออกด้วย

Reply:
Tip: you can use suggestion to ease a subsequent changes.

```suggestion
    <div className={`h-full p-4 p-8 text-white ${color}`}>
```